Terran-One/cw-simulate-ui
# Add option to delete contract

## Issue link

[WL-XXX](https://terran-one.atlassian.net/browse/WL-XXX)

## Description

Add an option to delete contracts. This doesn't delete the code from the immutable object. We can address that later. 

## Test steps

1. Upload a WASM file
2. Verify that delete contract works 
3.
